feat: add changelog for 2.0.0a13 (#17)
someengineering/resoto.com: news/2022-01-28-v2.0.0.0a13.md

# v2.0.0a13

> Starting wits this release Resoto ships as multiple independent Docker containers. See [packages](https://github.com/orgs/someengineering/packages?repo_name=resoto) for a list of available containers. The single container, which includes all dependencies still exist.

Reviewer: We should not link to the GitHub package list, since we want users to consume our packages via the Scarf Gateway.

Also, "with" is misspelled.

## Features

> - [[70ae7cc9]](https://github.com/someengineering/resoto/commit/70ae7cc9 ) _[docker]_ Split of Docker images ([#551](https://github.com/someengineering/resoto/pull/#551))
> - [[774d2c60]](https://github.com/someengineering/resoto/commit/774d2c60 ) _[docker]_ Build and push images to Docker Hub ([#475](https://github.com/someengineering/resoto/pull/#475))
> - [[6e2a68f3]](https://github.com/someengineering/resoto/commit/6e2a68f3 ) _[resotocore]_ Enable global search ([#591](https://github.com/someengineering/resoto/pull/#591))
> - [[270fedbd]](https://github.com/someengineering/resoto/commit/270fedbd ) _[resotocore]_ Create user/database with default setup ([#568](https://github.com/someengineering/resoto/pull/#568))
> - [[4dc167d9]](https://github.com/someengineering/resoto/commit/4dc167d9 ) _[resostocore]_ Add system info ([#561](https://github.com/someengineering/resoto/pull/#561))
> - [[8932d66c]](https://github.com/someengineering/resoto/commit/8932d66c ) _[resotocore]_ allow to set the default edge_type via the env ([#583](https://github.com/someengineering/resoto/pull/#583))

Reviewer: ```suggestion
- [[70ae7cc9]](https://github.com/someengineering/resoto/commit/70ae7cc9 ) _[docker]_ Split of Docker images ([#551](https://github.com/someengineering/resoto/pull/551))
- [[774d2c60]](https://github.com/someengineering/resoto/commit/774d2c60 ) _[docker]_ Build and push images to Docker Hub ([#475](https://github.com/someengineering/resoto/pull/475))
- [[6e2a68f3]](https://github.com/someengineering/resoto/commit/6e2a68f3 ) _[resotocore]_ Enable global search ([#591](https://github.com/someengineering/resoto/pull/591))
- [[270fedbd]](https://github.com/someengineering/resoto/commit/270fedbd ) _[resotocore]_ Create user/database with default setup ([#568](https://github.com/someengineering/resoto/pull/568))
- [[4dc167d9]](https://github.com/someengineering/resoto/commit/4dc167d9 ) _[resostocore]_ Add system info ([#561](https://github.com/someengineering/resoto/pull/561))
- [[8932d66c]](https://github.com/someengineering/resoto/commit/8932d66c ) _[resotocore]_ allow to set the default edge_type via the env ([#583](https://github.com/someengineering/resoto/pull/583))
```

## Fixes

> - [[a0e60516]](https://github.com/someengineering/resoto/commit/a0e60516 ) _[resotocore]_ Automatic merge queries also for aggregation variables and functions ([#590](https://github.com/someengineering/resoto/pull/#590))
> - [[5993df69]](https://github.com/someengineering/resoto/commit/5993df69 ) _[resotocore]_ tag command uses the ancestors data structure ([#579](https://github.com/someengineering/resoto/pull/#579))
> - [[04fe5ce4]](https://github.com/someengineering/resoto/commit/04fe5ce4 ) _[resotocore]_ Define configfile parameter explicitly ([#578](https://github.com/someengineering/resoto/pull/#578))
> - [[3e9d2c54]](https://github.com/someengineering/resoto/commit/3e9d2c54 ) _[resotocore]_ MergeTerm and NotTerm need to change variables ([#549](https://github.com/someengineering/resoto/pull/#549))
> - [[d3464ac8]](https://github.com/someengineering/resoto/commit/d3464ac8 ) _[resotocore]_ Unify output for boolean properties. ([#552](https://github.com/someengineering/resoto/pull/#552))
> - [[8eabbcad]](https://github.com/someengineering/resoto/commit/8eabbcad ) _[resotocore]_ Fix preliminary hangup of workers ([#577](https://github.com/someengineering/resoto/pull/#577))

Reviewer: ```suggestion
- [[a0e60516]](https://github.com/someengineering/resoto/commit/a0e60516 ) _[resotocore]_ Automatic merge queries also for aggregation variables and functions ([#590](https://github.com/someengineering/resoto/pull/590))
- [[5993df69]](https://github.com/someengineering/resoto/commit/5993df69 ) _[resotocore]_ tag command uses the ancestors data structure ([#579](https://github.com/someengineering/resoto/pull/579))
- [[04fe5ce4]](https://github.com/someengineering/resoto/commit/04fe5ce4 ) _[resotocore]_ Define configfile parameter explicitly ([#578](https://github.com/someengineering/resoto/pull/578))
- [[3e9d2c54]](https://github.com/someengineering/resoto/commit/3e9d2c54 ) _[resotocore]_ MergeTerm and NotTerm need to change variables ([#549](https://github.com/someengineering/resoto/pull/549))
- [[d3464ac8]](https://github.com/someengineering/resoto/commit/d3464ac8 ) _[resotocore]_ Unify output for boolean properties. ([#552](https://github.com/someengineering/resoto/pull/552))
- [[8eabbcad]](https://github.com/someengineering/resoto/commit/8eabbcad ) _[resotocore]_ Fix preliminary hangup of workers ([#577](https://github.com/someengineering/resoto/pull/577))
```

> ## Features
> 
> - [[70ae7cc9]](https://github.com/someengineering/resoto/commit/70ae7cc9 ) _[docker]_ Split of Docker images ([#551](https://github.com/someengineering/resoto/pull/551))
> - [[774d2c60]](https://github.com/someengineering/resoto/commit/774d2c60 ) _[docker]_ Build and push images to Docker Hub ([#475](https://github.com/someengineering/resoto/pull/475))
> - [[6e2a68f3]](https://github.com/someengineering/resoto/commit/6e2a68f3 ) _[resotocore]_ Enable global search ([#591](https://github.com/someengineering/resoto/pull/591))
> - [[270fedbd]](https://github.com/someengineering/resoto/commit/270fedbd ) _[resotocore]_ Create user/database with default setup ([#568](https://github.com/someengineering/resoto/pull/568))
> - [[4dc167d9]](https://github.com/someengineering/resoto/commit/4dc167d9 ) _[resostocore]_ Add system info ([#561](https://github.com/someengineering/resoto/pull/561))
> - [[8932d66c]](https://github.com/someengineering/resoto/commit/8932d66c ) _[resotocore]_ allow to set the default edge_type via the env ([#583](https://github.com/someengineering/resoto/pull/583))
> 
> ## Fixes
> 
> - [[a0e60516]](https://github.com/someengineering/resoto/commit/a0e60516 ) _[resotocore]_ Automatic merge queries also for aggregation variables and functions ([#590](https://github.com/someengineering/resoto/pull/590))
> - [[5993df69]](https://github.com/someengineering/resoto/commit/5993df69 ) _[resotocore]_ tag command uses the ancestors data structure ([#579](https://github.com/someengineering/resoto/pull/579))
> - [[04fe5ce4]](https://github.com/someengineering/resoto/commit/04fe5ce4 ) _[resotocore]_ Define configfile parameter explicitly ([#578](https://github.com/someengineering/resoto/pull/578))
> - [[3e9d2c54]](https://github.com/someengineering/resoto/commit/3e9d2c54 ) _[resotocore]_ MergeTerm and NotTerm need to change variables ([#549](https://github.com/someengineering/resoto/pull/549))
> - [[d3464ac8]](https://github.com/someengineering/resoto/commit/d3464ac8 ) _[resotocore]_ Unify output for boolean properties. ([#552](https://github.com/someengineering/resoto/pull/552))
> - [[8eabbcad]](https://github.com/someengineering/resoto/commit/8eabbcad ) _[resotocore]_ Fix preliminary hangup of workers ([#577](https://github.com/someengineering/resoto/pull/577))
> 
> ## Chore
> 
> - [[924d2912]](https://github.com/someengineering/resoto/commit/924d2912 ) _[resotolib]_ Update metrics queries to use new ancestors section ([#596](https://github.com/someengineering/resoto/pull/596))
> - [[f5963c7f]](https://github.com/someengineering/resoto/commit/f5963c7f ) _[resotocore]_ Rename edge type dependency to default ([#585](https://github.com/someengineering/resoto/pull/585))
> - [[d4e96089]](https://github.com/someengineering/resoto/commit/d4e96089 ) _[resotocore]_ graph update expect the replace flag in the metadata section ([#582](https://github.com/someengineering/resoto/pull/582))
> - [[4320a935]](https://github.com/someengineering/resoto/commit/4320a935 ) _[resotocore]_ Remove section aware query API ([#564](https://github.com/someengineering/resoto/pull/564))
> - [[ba05ba5d]](https://github.com/someengineering/resoto/commit/ba05ba5d ) _[resoto-ui]_ Move UI to dedicated repo ([#563](https://github.com/someengineering/resoto/pull/563))
> - [[79a4181c]](https://github.com/someengineering/resoto/commit/79a4181c ) _[resotocore]_ Use proper time units ([#559](https://github.com/someengineering/resoto/pull/559))

Reviewer: ```suggestion
## Features

- [`70ae7cc9`](https://github.com/someengineering/resoto/commit/70ae7cc9) <span class="badge badge--info">docker</span> Split of Docker images ([#551](https://github.com/someengineering/resoto/pull/551))
- [`774d2c60`](https://github.com/someengineering/resoto/commit/774d2c60) <span class="badge badge--info">docker</span> Build and push images to Docker Hub ([#475](https://github.com/someengineering/resoto/pull/475))
- [`6e2a68f3`](https://github.com/someengineering/resoto/commit/6e2a68f3) <span class="badge badge--primary">resotocore</span> Enable global search ([#591](https://github.com/someengineering/resoto/pull/591))
- [`270fedbd`](https://github.com/someengineering/resoto/commit/270fedbd) <span class="badge badge--primary">resotocore</span> Create user/database with default setup ([#568](https://github.com/someengineering/resoto/pull/568))
- [`4dc167d9`](https://github.com/someengineering/resoto/commit/4dc167d9) <span class="badge badge--primary">resotocore</span> Add system info ([#561](https://github.com/someengineering/resoto/pull/561))
- [`8932d66c`](https://github.com/someengineering/resoto/commit/8932d66c) <span class="badge badge--primary">resotocore</span> allow to set the default edge_type via the env ([#583](https://github.com/someengineering/resoto/pull/583))

## Fixes

- [`a0e60516`](https://github.com/someengineering/resoto/commit/a0e60516) <span class="badge badge--primary">resotocore</span> Automatic merge queries also for aggregation variables and functions ([#590](https://github.com/someengineering/resoto/pull/590))
- [`5993df69`](https://github.com/someengineering/resoto/commit/5993df69) <span class="badge badge--primary">resotocore</span> tag command uses the ancestors data structure ([#579](https://github.com/someengineering/resoto/pull/579))
- [`04fe5ce4`](https://github.com/someengineering/resoto/commit/04fe5ce4) <span class="badge badge--primary">resotocore</span> Define configfile parameter explicitly ([#578](https://github.com/someengineering/resoto/pull/578))
- [`3e9d2c54`](https://github.com/someengineering/resoto/commit/3e9d2c54) <span class="badge badge--primary">resotocore</span> MergeTerm and NotTerm need to change variables ([#549](https://github.com/someengineering/resoto/pull/549))
- [`d3464ac8`](https://github.com/someengineering/resoto/commit/d3464ac8) <span class="badge badge--primary">resotocore</span> Unify output for boolean properties ([#552](https://github.com/someengineering/resoto/pull/552))
- [`8eabbcad`](https://github.com/someengineering/resoto/commit/8eabbcad) <span class="badge badge--primary">resotocore</span> Fix preliminary hangup of workers ([#577](https://github.com/someengineering/resoto/pull/577))

## Chores

- [`924d2912`](https://github.com/someengineering/resoto/commit/924d2912) <span class="badge badge--primary">resotolib</span> Update metrics queries to use new ancestors section ([#596](https://github.com/someengineering/resoto/pull/596))
- [`f5963c7f`](https://github.com/someengineering/resoto/commit/f5963c7f) <span class="badge badge--primary">resotocore</span> Rename edge type dependency to default ([#585](https://github.com/someengineering/resoto/pull/585))
- [`d4e96089`](https://github.com/someengineering/resoto/commit/d4e96089) <span class="badge badge--primary">resotocore</span> graph update expect the replace flag in the metadata section ([#582](https://github.com/someengineering/resoto/pull/582))
- [`4320a935`](https://github.com/someengineering/resoto/commit/4320a935) <span class="badge badge--primary">resotocore</span> Remove section aware query API ([#564](https://github.com/someengineering/resoto/pull/564))
- [`ba05ba5d`](https://github.com/someengineering/resoto/commit/ba05ba5d) <span class="badge badge--primary">resoto-ui</span> Move UI to dedicated repo ([#563](https://github.com/someengineering/resoto/pull/563))
- [`79a4181c`](https://github.com/someengineering/resoto/commit/79a4181c) <span class="badge badge--primary">resotocore</span> Use proper time units ([#559](https://github.com/someengineering/resoto/pull/559))```

> - [[924d2912]](https://github.com/someengineering/resoto/commit/924d2912 ) _[resotolib]_ Update metrics queries to use new ancestors section ([#596](https://github.com/someengineering/resoto/pull/#596))
> - [[f5963c7f]](https://github.com/someengineering/resoto/commit/f5963c7f ) _[resotocore]_ Rename edge type dependency to default ([#585](https://github.com/someengineering/resoto/pull/#585))
> - [[d4e96089]](https://github.com/someengineering/resoto/commit/d4e96089 ) _[resotocore]_ graph update expect the replace flag in the metadata section ([#582](https://github.com/someengineering/resoto/pull/#582))
> - [[4320a935]](https://github.com/someengineering/resoto/commit/4320a935 ) _[resotocore]_ Remove section aware query API ([#564](https://github.com/someengineering/resoto/pull/#564))
> - [[ba05ba5d]](https://github.com/someengineering/resoto/commit/ba05ba5d ) _[resoto-ui]_ Move UI to dedicated repo ([#563](https://github.com/someengineering/resoto/pull/#563))
> - [[79a4181c]](https://github.com/someengineering/resoto/commit/79a4181c ) _[resotocore]_ Use proper time units ([#559](https://github.com/someengineering/resoto/pull/#559))

Reviewer: ```suggestion
- [[924d2912]](https://github.com/someengineering/resoto/commit/924d2912 ) _[resotolib]_ Update metrics queries to use new ancestors section ([#596](https://github.com/someengineering/resoto/pull/596))
- [[f5963c7f]](https://github.com/someengineering/resoto/commit/f5963c7f ) _[resotocore]_ Rename edge type dependency to default ([#585](https://github.com/someengineering/resoto/pull/585))
- [[d4e96089]](https://github.com/someengineering/resoto/commit/d4e96089 ) _[resotocore]_ graph update expect the replace flag in the metadata section ([#582](https://github.com/someengineering/resoto/pull/582))
- [[4320a935]](https://github.com/someengineering/resoto/commit/4320a935 ) _[resotocore]_ Remove section aware query API ([#564](https://github.com/someengineering/resoto/pull/564))
- [[ba05ba5d]](https://github.com/someengineering/resoto/commit/ba05ba5d ) _[resoto-ui]_ Move UI to dedicated repo ([#563](https://github.com/someengineering/resoto/pull/563))
- [[79a4181c]](https://github.com/someengineering/resoto/commit/79a4181c ) _[resotocore]_ Use proper time units ([#559](https://github.com/someengineering/resoto/pull/559))
```

# v2.0.0a13

> Starting with this release Resoto ships as multiple independent Docker containers. The single container, which includes all dependencies still exist.

Reviewer: ```suggestion
Starting with this release, Resoto ships with separate Docker images for each of its components:

- `somecr.io/someengineering/resotocore`
- `somecr.io/someengineering/resotoworker`
- `somecr.io/someengineering/resotoshell`
- `somecr.io/someengineering/resotometrics`

These images allow for greater flexibility in deploying Resoto, and also make it possible to independently update individual components.

However, `somecr.io/someengineering/resoto` (which packages all dependencies into a single image) is still available.
```
